### PR TITLE
cleanup(repo): implement sort-package-json into git hooks

### PR DESCRIPTION
**Pull request by benpsnyder**
## Current Behavior

The package.json file is not sorted automatically

## Expected Behavior

Running `npx sort-package-json` before every push will keep package.json tidy

## Related Issue(s)

Fixes #

### CODE REVIEW

File: .husky/pre-push [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+npx npx sort-package-json
```
Comment (pan93412) [inline .husky/pre-push:4]: Seems there is an additional `npx`. Should it be required?

Comment (benpsnyder) [reply .husky/pre-push]: Good catch .... just fixed it

**Comment by pan93412** [comment]
The correct form of a commit message should be:

        fix(core): brief description

You can edit it by:

        git commit --amend

**Comment by benpsnyder** [comment]
> The correct form of a commit message should be:
> 
> ```
> fix(core): brief description
> ```
> 
> and your scope and type are in a wrong order. You can edit it by:
> 
> ```
> git commit --amend
> ```

Nice catch. fixed...

**Comment by FrozenPandaz** [comment]
Sorry, we're not interested in a solution during the pre push step. If it were a prettier plugin that would be better. Would you like to open a new PR that implements a prettier plugin that sorts package.json?

**Pull request closed without merging**